services/ui-src/src/utils/scroll/scrollToTop.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import { animateScroll } from "react-scroll";
+
+export const ScrollToTopComponent = () => {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    scrollOnRouteChange();
+  }, [pathname]);
+
+  return null;
+};
+
+const scrollOnRouteChange = () => {
+  animateScroll.scrollToTop({
+    duration: 0,
+  });
+};
+
+export const scrollToTopFormSubmit = () => {
```

Comment:
If we are keeping the user on the same page post submit, should we be scrolling the user to the top? It seems like unexpected behavior. If we are bouncing them back to the top (like old form submits that had to hit a backend server and then reload the page), I think we are also fine with no animation here and just setting them to top of page (just like the page reloaded).

Reply:
Hmm... interesting thought. I agree that we definitely need to put them at the top of the page so it's evident that the operation succeeded. But it doesn't necessarily have to be a scroll and we could just set them to the top. Might be something to run past the designers.

Reply:
We can test it out and adjust as needed. I'm good the current animated scroll.